xml/System/DateOnly.xml

```diff
@@ -1,2442 +1,2445 @@
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Attributes>
+        <Attribute FrameworkAlternate="net-8.0">
+          <AttributeName Language="C#">[System.Runtime.CompilerServices.NullableContext(1)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.Runtime.CompilerServices.NullableContext(1)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <Parameters>
+        <Parameter Name="year" Type="System.Int32" />
+        <Parameter Name="month" Type="System.Int32" />
+        <Parameter Name="day" Type="System.Int32" />
+        <Parameter Name="calendar" Type="System.Globalization.Calendar" />
+      </Parameters>
+      <Docs>
+        <param name="year">The year (1 through the number of years in calendar).</param>
+        <param name="month">The month (1 through the number of months in calendar).</param>
+        <param name="day">The day (1 through the number of days in <paramref name="month" />).</param>
+        <param name="calendar">The calendar that is used to interpret year, month, and day.<paramref name="month" />.</param>
+        <summary>Creates a new instance of the <see cref="T:System.DateOnly" /> structure to the specified year, month, and day for the specified calendar.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="AddDays">
+      <MemberSignature Language="C#" Value="public DateOnly AddDays (int value);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig instance valuetype System.DateOnly AddDays(int32 value) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.DateOnly.AddDays(System.Int32)" />
+      <MemberSignature Language="VB.NET" Value="Public Function AddDays (value As Integer) As DateOnly" />
+      <MemberSignature Language="F#" Value="member this.AddDays : int -&gt; DateOnly" Usage="dateOnly.AddDays value" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; DateOnly AddDays(int value);" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Runtime</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+        <AssemblyVersion>8.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.DateOnly</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="value" Type="System.Int32" />
+      </Parameters>
+      <Docs>
+        <param name="value">The number of days to add. To subtract days, specify a negative number.</param>
+        <summary>Adds the specified number of days to the value of this instance.</summary>
+        <returns>An instance whose value is the sum of the date represented by this instance and the number of days represented by value.</returns>
+        <exception cref="ArgumentOutOfRangeException">

```

Comment:
Please do it as it is done in https://github.com/dotnet/dotnet-api-docs/blob/173aa811f0080f5573c370e43852c1c61bed18d0/xml/System/DateTime.xml#L977. 

Also, please revert any white spaces changes in the file you are editing. Thanks!

Reply:
Sorry, I was just using the GitHub web editor for this, so it must have been making all the whitespace changes...

Reply:
No worries. I'll wait for your update and then we can proceed. Thanks!

Reply:
I'm going to send another pull request in order to avoid Git-history trashing.